solidity/contracts/api/IBondedECDSAKeepFactory.sol

```diff
@@ -22,4 +22,12 @@ interface IBondedECDSAKeepFactory {
     /// @notice Gets a fee estimate for opening a new keep.
     /// @return Uint256 estimate.
     function openKeepFeeEstimate() external view returns (uint256);
+
+    /// @notice Gets the total weight of operators
+    /// in the sortition pool for the given application.
+    /// @param _application Address of the application.
+    /// @return The sum of all registered operators' weights in the pool.
+    function getSortitionPoolWeight(
+        address _application
+    ) external view returns (uint256 poolWeight);
```

Comment:
Let's just do `returns (uint256)`, the function is `getSortitionPoolWeight` so it's clear what are we returning here. It's the same case as with `openKeepFeeEstimate`